plugins/org.eclipse.n4js.generator.common/src/org/eclipse/n4js/generator/common/AbstractSubGenerator.xtend

```diff
@@ -101,25 +101,14 @@ abstract class AbstractSubGenerator implements ISubGenerator {
 			internalDoGenerate(input, GeneratorOption.DEFAULT_OPTIONS, fsa);
 		} catch (Exception e) {
```

Comment:
As far as I understand, the idea is to always use the methods provided by `OperationCanceledManager`. Having a dedicated catch would boil down to using
```
e instanceof OperationCanceledException
```
instead of
```
operationCanceledManager.isOperationCanceledException(e)
```